snuba/settings.py

```diff
@@ -80,9 +81,19 @@
     # See snuba/utils/streams/backends/kafka.py for the supported options
     "bootstrap.servers": os.environ.get("DEFAULT_BROKERS", "localhost:9092"),
 }
+
+# DEPRECATED, please use KAFKA_BROKER_CONFIG instead
 STORAGE_BROKER_CONFIG: Mapping[str, Mapping[str, Any]] = {}
+
+# DEPRECATED, please use KAFKA_TOPIC_MAP instead
 STORAGE_TOPICS: Mapping[str, Mapping[str, Any]] = {}
 
+# Mapping of default Kafka topic name to custom names
+KAFKA_TOPIC_MAP: Mapping[str, str] = {}
+
+# Mapping of default Kafka topic name to broker config
```

Comment:
Is this the logical topic name (the one we use in the code base) or the concrete topic name ?

Reply:
It's the logical topic name. Note that is inconsistent with TOPIC_PARTITION_COUNTS which will look for the concrete name. Personally I think this way is a better default and we should migrate TOPIC_PARTITION_COUNTS to use the logical name. I left a TODO in the codebase about it. Let me know if you have a different opinion though.

Reply:
Yeah the logical topic is better. I actually don't even think we should explicitly state the number of partitions in the settings, but that is another topic altogether.